src/Shell/Task/BeditaPluginTask.php

```diff
@@ -0,0 +1,17 @@
+<?php
+namespace BEdita\App\Shell\Task;
+
+use Bake\Shell\Task\PluginTask as BakeTask;
+
+/**
+ * The BEdita Plugin Task handles creating an empty BEdita4 plugin, ready to be used
+ */
+class BeditaPluginTask extends BakeTask
+{
+    /**
+     * {@inheritDoc}
+     */
+    protected function _modifyBootstrap($plugin, $hasAutoloader)
+    {
```

Comment:
I'd add a line of comment to explain why this method is being overwritten empty. Something like `// Avoid updating bootstrap.php` would be just fine IMHO.